CHANGELOG.md

```diff
@@ -42,6 +42,7 @@
 - [vsx-registry] improved styling of the `Extensions` view [#11494](https://github.com/eclipse-theia/theia/pull/11494)
 - [vsx-registry] removed localization for `Open VSX Registry` [#11523](https://github.com/eclipse-theia/theia/pull/11523)
 - [vsx-registry] updated extension editor rendering [#11605](https://github.com/eclipse-theia/theia/pull/11605)
+- [custom-editor] Auto Save on Binary Custom Editor does not work correctly [#11460](https://github.com/eclipse-theia/theia/issues/11460)
```

Comment:
minor: `1.29.0` was released last week, we'd need to add it under a new `1.30.0` entry. 
You can remove it for now, I generally update the changelog prior to the release anyways.